src/openseadragon.js

```diff
@@ -816,6 +816,32 @@ window.OpenSeadragon = window.OpenSeadragon || function( options ){
                     canvasElement.getContext( '2d' ) );
     }());
 
+    /**
+     * @private
+     * @inner
+     * @function
+     * @returns {Number} The device's pixel density ratio, or null if canvas isn't supported by the browser.
```

Comment:
Also, I suppose this should really say something about how it's the pixel density ratio in relation to canvas backing store.